app/scripts/modules/titus/src/serverGroup/details/rollback/rollbackServerGroup.controller.js

```diff
@@ -89,7 +89,9 @@ module.exports = angular
 
         if (rollbackType === 'PREVIOUS_IMAGE') {
           // no need to validate when using an explicit image
-          return true;
+          // return true;
+          // KLUDGE: temporarily disabling previous image rollbacks for titus because they do not work properly
+          return false;
```

Comment:
I understand that this is a safety issue.  Will users be confused by this?  I suppose there's no feedback why the rollback is invalid.

Reply:
Yeah.. agree. I was trying to come up with the right text/place. Think I got something that works.